warehouse/query-core/src/main/java/datawave/query/jexl/visitors/RegexIndexExpansionVisitor.java

```diff
@@ -718,6 +764,21 @@ protected void rebuildFutureJexlNode(FutureJexlNode futureJexlNode) {
             }
         }
 
+        if (lookupCache != null) {
+            String field = JexlASTHelper.getIdentifier(currentNode);
+            if (fieldsToTerms.containsKey(field) && (fieldsToTerms.get(field).isThresholdExceeded() || fieldsToTerms.get(field).isEmpty())) {
+                LookupCacheKey key = getKey(currentNode);
+                lookupCache.put(key, false);
+            }
+        }
+
```

Comment:
this can move up into BaseIndexExpansionVisitor with a stub for getKey() to be implemented in each impl